package.json

```diff
@@ -45,6 +45,7 @@
     "babel-eslint": "^7.0.0",
     "benchmark": "~2.1.0",
     "browserify": "^13.0.0",
+    "canvas": "^1.6.2",
```

Comment:
NOTE: this is a relatively nontrivial dep for something just used in one test…discuss